tests/testthat/test_random.r

```diff
@@ -6,20 +6,22 @@ genome <- tibble::tribble(
   "chr2", 1e6
 ) %>% mutate(chrom = as.character(chrom))
 
+seed <- 1010486
```

Comment:
Can you define this at the top of the file, and add a comment i.e. "seed for reproducible bed_random tests'